packages/ignite-dev-screens/index.js

```diff
@@ -1,27 +1,4 @@
-const copyDevScreens = async function (context) {
-  // grab some features
-  const { ignite } = context
-
-  const screens = ['APITestingScreen', 'ComponentExamplesScreen', 'DeviceInfoScreen', 'DevscreensButton', 'PluginExamplesScreen', 'PresentationScreen', 'ThemeScreen', 'ButtonBox']
-
-  const jobs = []
-
-  screens.map((screen) => {
-    jobs.push(
-      {
-        template: `${screen}.js.ejs`,
-        target: `ignite/DevScreens/${screen}.js`
-      },
-      {
-        template: `${screen}Style.js.ejs`,
-        target: `ignite/DevScreens/Styles/${screen}Style.js`
-      }
-    )
-  })
-
-  // make the templates
-  await ignite.copyBatch(context, jobs, {})
-}
+const sourceFolder = `${process.cwd()}/node_modules/ignite-dev-screens/templates`
```

Comment:
It'll always be the root of your ignited app.  In fact I think for most operations, you replace `${process.cwd()}/` with `./` or hopefully even `` (blank).

Symlinks are the only think I would imagine would shit the bed.  I don't think we're using symlinks yet.

`SOURCE_PATH` still is ambiguous to me since we're in the business of breeding plugins.

I will agree with you `process.cwd()` makes me stop & think sometimes too.  Someone should see if it's optional.